vector/src/main/scala/geotrellis/vector/SpatialIndex.scala

```diff
@@ -81,10 +87,20 @@ class SpatialIndex[T](val measure: Measure = Measure.Euclidean) extends Serializ
         }
         rtree.query(extent.jtsEnvelope, visitor)
       }
+
+      // Traversable implementations must override iterator in 2.13
+      def iterator: Iterator[T] =
+        rtree.query(extent.jtsEnvelope).asScala.map(_.asInstanceOf[T]).iterator
+    }
+
+  def pointsInExtentAsIterable(extent: Extent): Iterable[T] =
```

Comment:
Tbh I like this function naming much more than the `{traverse|iterate}PointsInExtent`. :+1: